README.md

```diff
@@ -4,23 +4,43 @@
 [![Docs.rs](https://docs.rs/mini_gl_fb/badge.svg)](https://docs.rs/mini_gl_fb)
 
 Mini GL Framebuffer provides an easy way to draw to a window from a pixel buffer. OpenGL
-alternative to other easy framebuffer libraries.
+alternative to other easy framebuffer libraries like `minifb` and `pixels`.

```

Comment:
Note to self, add links later to be a cool cat in the ecosystem

Should we spell out differences more too later? (Not necessary for this PR to be merged)

Reply:
> Should we spell out differences more too later?

Could add a feature matrix in the docs, if you like.